src/cryptojwt/key_jar.py

```diff
@@ -32,7 +32,6 @@ def __init__(
         remove_after=3600,
         httpc=None,
         httpc_params=None,
-        storage=None,
```

Comment:
Can we assume that no one (excluding closing family) has used the storage class yet?

Reply:
Yes !